### PR TITLE
TELCODOCS-1077: Release note for adding LVMS via ZTP

### PR DESCRIPTION
**Pull request by amolnar-gh**
Version(s): 4.13

Issue: https://issues.redhat.com/browse/TELCODOCS-1077

Link to docs preview: https://57660--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-13-release-notes.html#ocp-4-13-storage-lvms-in-zpt

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-13-release-notes.adoc [hunk 1]
```diff
@@ -208,6 +208,11 @@ For more information, see xref:../networking/hardware_networks/switching-bf2-nic
 [id="ocp-4-13-storage"]
 === Storage
 
+[id="ocp-4-13-storage-lvms-in-zpt"]
+==== Support for {lvms} in GitOps ZTP
+
+In {product-title} {product-version}, you can add and configure {lvms-first} through GitOps ZTP. For more information, see xref:../scalability_and_performance/ztp_far_edge/ztp-manual-install.adoc#ztp-lvms-installing-using-web-console_ztp-manual-install[Intalling LVM Storage by using the web console] and xref:../scalability_and_performance/ztp_far_edge/ztp-reference-cluster-configuration-for-vdu.adoc#lvms-configuring-lvms-on-sno_sno-configure-for-vdu[LVM Storage].
```
Comment (aireilly) [inline release_notes/ocp-4-13-release-notes.adoc:214]: Will also need a link to the actual ZTP procedure: https://github.com/openshift/openshift-docs/pull/57467

File: release_notes/ocp-4-13-release-notes.adoc [hunk 2]
```diff
@@ -289,6 +289,11 @@ The cluster network can be expanded to support the addition of nodes to the clus
 
 In {product-title} {product-version}, {lvms} is supported in dual-stack for IPv4 and IPv6 network environments. For more information, see xref:../networking/ovn_kubernetes_network_provider/converting-to-dual-stack.adoc#nw-dual-stack-convert_converting-to-dual-stack[Converting to a dual-stack cluster network].
 
+[id="ocp-4-13-storage-lvms-in-zpt"]
+==== Support for {lvms} in GitOps ZTP
+
+In {product-title} {product-version}, you can add and configure {lvms-first} through GitOps ZTP. For more information, see xref:../scalability_and_performance/ztp_far_edge/ztp-manual-install.adoc#ztp-lvms-installing-using-web-console_ztp-manual-install[Intalling LVM Storage by using the web console], xref:../scalability_and_performance/ztp_far_edge/ztp-advanced-policy-config.adoc#ztp-provisioning-lvm-storage_ztp-advanced-policy-config[Configuring LVM Storage using PolicyGenTemplate CRs], and xref:../scalability_and_performance/ztp_far_edge/ztp-reference-cluster-configuration-for-vdu.adoc#lvms-configuring-lvms-on-sno_sno-configure-for-vdu[LVM Storage].
```
Comment (lahinson) [inline release_notes/ocp-4-13-release-notes.adoc:295]: Two of the links don't seem to match the link text that you mention.

- The first link, `xref:../scalability_and_performance/ztp_far_edge/ztp-manual-install.adoc#ztp-lvms-installing-using-web-console_ztp-manual-install`, opens a page entitled "Manually installing a single-node OpenShift cluster with ZTP".

- The third link, `xref:../scalability_and_performance/ztp_far_edge/ztp-reference-cluster-configuration-for-vdu.adoc#lvms-configuring-lvms-on-sno_sno-configure-for-vdu`, opens a page entitled "Recommended single-node OpenShift cluster configuration for vDU application workloads".

File: release_notes/ocp-4-13-release-notes.adoc
Comment (lahinson) [inline release_notes/ocp-4-13-release-notes.adoc:1]: Did QE review and approve the text? The QE checkbox isn't selected.

**Review by lahinson — COMMENTED** [review]
Just a couple of questions -- mainly about links. Let me know if you have questions for me. Thanks!

File: release_notes/ocp-4-13-release-notes.adoc (hunk 2 above)
Comment (amolnar-rh) [reply release_notes/ocp-4-13-release-notes.adoc:360]: Now, links are working as intended

**Comment by stevsmit** [comment]
lgtm

**Pull request merged**